### PR TITLE
Implement track endpoint and add fields

### PR DESCRIPTION
**Pull request by veidenberg**
This PR:
* adds `track/:track_id` enpoint
* replaces all track IDs with UUIDs
* adds `type` and `datafiles` fields to tracks
* maintenance (bump requirements, improve datamodel, readme etc.)

Ticket: https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2168

Example API requests:
* http://update.review.ensembl.org/api/tracks/track/5406535e-b21a-45a6-b9d6-75482353206f
* http://update.review.ensembl.org/api/tracks/track_categories/a7335667-93e7-11ec-a39d-005056b38ce3

### CODE REVIEW

**Comment by veidenberg** [comment]
This review app uses dev database with all 6 variation tracks enabled for human. The 5 extra tracks will be removed from datafile before merging to dev to have single variation track for all species in production db (used in staging, internal, live).

**Comment by azangru** [comment]
I wonder if there is a clean way of not including the datafiles field in the response to the client. The client doesn't care.

**Comment by veidenberg** [comment]
> I wonder if there is a clean way of not including the datafiles field in the response to the client. The client doesn't care.

Done. Likewise, I've excluded some fields from the track payload for GB (`additional_info`, `description`, `sources`).

File: tracks/serializers.py [hunk 1]
```diff
@@ -11,17 +11,24 @@ class Meta:
...
+class ClientTrackSerializer(serializers.ModelSerializer):
     sources = SourceSerializer(many=True, read_only=True)
 
     class Meta:
         model = Track
-        fields = ["label", "track_id", "colour", "trigger", "display_order", "on_by_default", "additional_info", "description", "sources"]
+        fields = ["label", "track_id", "colour", "trigger", "type",
+            "display_order", "on_by_default", "additional_info", "description", "sources"]
+
+class BackendTrackSerializer(serializers.ModelSerializer):
```
Comment (kamaldodiya) [inline tracks/serializers.py:22]: We are already in Backend. If this very specific to GenomeBrowser then GenomeBrowserTrackSerialiser or something would be more appropriate.

**Pull request merged**